tools/SetupFlow/DevHome.SetupFlow/DevHome.SetupFlow.csproj

```diff
@@ -18,6 +18,7 @@
     <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0182" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.1" />
     <PackageReference Include="Microsoft.Internal.Windows.DevHome.Helpers" Version="1.0.*-*" />
+    <PackageReference Include="Microsoft.Management.Infrastructure" Version="2.0.0" />

```

Comment:
Because you reference this package in SetupFlow.Common, I think you don't need to include it here.